data/phone/text/irwin_caller.asm

```diff
@@ -259,7 +271,7 @@ UnknownText_0x64881:
 	line "a good match!"
 	done
 
-UnknownText_0x648dc:
+IrwinNoseyText:
```

Comment:
It's "nosy". Also, that's more judging than descriptive; consider `IrwinSoMuchToChatAboutText`, `IrwinHowAreYouText`, `IrwinManyQuestionsText`.